packages/analytics-browser-test/test/index.test.ts

```diff
@@ -24,10 +24,10 @@ describe('integration', () => {
     test('should track event', async () => {
       const scope = nock(url).post(path).reply(200, success);
 
-      amplitude.init('API_KEY', undefined, {
+      await amplitude.init('API_KEY', undefined, {
         ...opts,
-      });
-      const response = await amplitude.track('test event', {
+      }).promise;
```

Comment:
added await on init here if not, init has not finished yet before the first track event is processed